test/main.go

```diff
@@ -78,10 +78,12 @@ func main() {
 		if err != nil {
 			testCluster.Shutdown()
 			log.Println("could not build flynn: ", err)
+			os.RemoveAll(rootFS)
 			return
 		}
 		if args.KeepRootFS {
 			fmt.Println("Built Flynn in rootfs:", rootFS)
+			os.Exit(0)
```

Comment:
The current flag name/definition does not imply this:

```text
-keep-rootfs don't remove the rootfs which was built to run the tests
```

Change the flag if this behavior makes sense.